yarn-project/end-to-end/Earthfile

```diff
@@ -5,7 +5,7 @@ E2E_COMPOSE_TEST:
   ARG test
   ARG compose_file=./scripts/docker-compose.yml
   ARG debug="aztec:*"
-  ARG EARTHLY_TARGET_NAME
+  ARG skip_build
```

Comment:
The usual ways to implicitly pass this through like ARG --global dont work with functions, and making this a parametric target instead of a function would make logging less nice (show the utility name instead of the test name), so living with a bit of passing this through the targets

Reply:
I know I've been complaining about unclear args rules, but shouldn't global args work with functions? According to [their docs](https://docs.earthly.dev/docs/guides/functions#scope) a function should be able to access global args.

Reply:
Ah thanks! I must have gotten tripped up. I'll try to simplify this